### ISSUE TITLE
Not able to run this template

### ISSUE THREAD

**Issue by maxmarkus** [gzuidhof/sunder-worker-template#9]
Hey Guido,
appreciate your effort on SunderJS and this template, which made me wanting to try it. 

Unfortunately I am stuck with several errors (using node 16 lts on MacOS Big Sur), right after fresh template clone and running `npm install`:
1. Following the readme and directly running `npm run miniflare` and `npm run watch` both show *ENOENT dist/index.mjs not found* which is true, since there is no dist folder yet.
2. Ran `npm run build` which generated the dist/index.mjs 
3. Ran `npm run miniflare` again which then shows:
```
import manifestJSON from "__STATIC_CONTENT_MANIFEST";
^^^^^^
SyntaxError: Cannot use import statement outside a module
    at new Script (node:vm:100:7)
    at VMScriptRunner.runAsScript (file:///..../sunder-worker-template/node_modules/@miniflare/runner-vm/src/index.ts:25:20)
    at VMScriptRunner.run (file://..../sunder-worker-template/node_modules/@miniflare/runner-vm/src/index.ts:89:12)
    at EventTarget.#reload
```

Some advice what I am doing wrong here?

Thanks, Markus

**Comment by gzuidhof** [comment]
Hi Markus,

I tried it on my Apple Silicon macbook (also Node v0.16.13), where I saw the exact same issue. The problem is the lack of the `wrangler.toml` because this repo only contains `wrangler.example.toml` (it makes dev of this template a bit easier), so you are supposed to rename that file. Miniflare doesn't complain as it uses defaults instead.

But this was not obviously documented at all, so I am going to fix that: I'll make sure in newer versions there is a `wrangler.toml` out of the box.
